Source/Scene/Cesium3DTileset.js

```diff
@@ -272,6 +226,50 @@ define([
         }
     });
 
+    Cesium3DTileset.prototype.loadTilesJson = function(tilesJson, parentTile) {
```

Comment:
Add a comment above to document this as `@private` since it won't be part of the public API.